### ISSUE TITLE
Are your writes atomic?

### ISSUE THREAD

**Issue by kvz** [feross/hostile#12]
Hi, 

I hosed my `/ets/hosts` file after using hostile in combination with `async.parallel`:

```coffeescript
    async.parallel [
      (cb) ->
        exe "sudo hostile set 127.0.1.1 api2-vbox.example.com", cb
      (cb) ->
        exe "sudo hostile set 127.0.1.1 tmp-vbox.example.com", cb
      (cb) ->
        exe "sudo hostile set 127.0.1.1 worker-vbox.example.com", cb
      (cb) ->
        exe "sudo hostile set 127.0.1.1 vbox.example.com", cb
    ], callback
```

Are you using something like https://github.com/npm/fs-write-stream-atomic to make sure writes are atomic?

(I could easily recover, but others might have bigger problems - and node makes it pretty easy to have things going on in parallel ofc)

**Comment by refactorized** [comment]
If the writes are not atomic, and the callback must be fired before calling it again, then I don't think there should be asynchronous methods to do this.  

I suppose it could be argued that a single asynchronous call would be slightly more performant than the synchronous one, but multiple calls must have some device set in place that enforces sequence, which is going to be more overhead than the sequenced calls.

In either case, until it is fixed, there should definitely be some verbiage on the readme regarding this aspect of the library.

**Comment by feross** [comment]
Duplicate of #9.
